test/fixtures/test-runner/output/hooks.snapshot

```diff
@@ -56,7 +56,7 @@ not ok 2 - before throws
   duration_ms: *
   type: 'suite'
   failureType: 'hookFailed'
-  error: 'failed running before hook'
+  error: 'before'
```

Comment:
If it looks like these error messages are actually less helpful - these are the actual message from the error objects thrown in the test.

Reply:
@cjihrig and why is there no stack? 😅

Reply:
Is that just the snapshot?